### DISCUSSION TITLE
Speechly Free Tier is no longer available.

### DISCUSSION THREAD

**Discussion by jdominguezs** [speechly/speechly#338] [General]
My applications have stopped working and appear as "Inactive," and my dashboard shows the message "Speechly Free Tier is no longer available." I have not received a notification about this issue. Apart from this, there is no way to upgrade to a paid plan or contact them. Please help.

**Comment by teelisyys** [comment]
Hi @jdominguezs ! Thank you for reaching out!

Speechly Free tier product is unfortunately no longer available. This change in product offering is due to [Speechly joining Roblox](https://www.speechly.com/blog/speechly-is-joining-roblox), and we announced it to our customers back in September by email. We are sorry that this announcement did not reach you.

Please be aware that we are not upgrading customers from Free tier to Professional or Enterprise tiers anymore either.

We hope that you can find a suitable replacement for Speechly Free tier product from some other voice technology provider. We apologize for the inconvenience and wish you all the best in your future endeavors.

We want to once again thank you, and all other developers who have trusted Speechly for voice technology needs, and our customers who have been with us through this journey.
